apps/admin-gui/src/app/admin/pages/admin-page/admin-audit-log/admin-audit-log.component.html

```diff
@@ -1,3 +1,19 @@
 <h1 class="page-subtitle">{{'ADMIN.AUDIT_LOG.TITLE'|translate}}</h1>
-<perun-web-apps-refresh-button (refresh)="refreshTable()"></perun-web-apps-refresh-button>
-<app-audit-messages-list [refresh]="refresh" [tableId]="tableId"></app-audit-messages-list>
+<div class="align-elements">
+  <perun-web-apps-refresh-button (refresh)="refreshTable()"></perun-web-apps-refresh-button>
+  <perun-web-apps-entity-search-select
+    [entities]="eventOptions"
+    [searchFunction]="searchFunction"
+    (entitySelected)="toggleEvent($event)"
+    [mainTextFunction]="mainTextFunction"
+    [secondaryTextFunction]="secondaryTextFunction"
+    [multiple]="true"
+    [disableAutoSelect]="true"
+    [fullWidth]="false"
+    [moveSelectedUp]="true"
+    [selectPlaceholder]="'ADMIN.AUDIT_LOG.SEARCH_DESCRIPTION'|translate"></perun-web-apps-entity-search-select>
```

Comment:
Entity search select is the base class for search select components. We don't use it directly but rather create a specification e.g. vo search select. I would like to ask if you could please create such a specification also for audit events.